.gitignore

```diff
@@ -7,3 +7,12 @@ venv/
 
 # Ignore VSCode data if present
 .vscode/
+
+# Ignore Emacs and Vim data if present
+*.swp
+*~
+
+# Rust
+rust/fourhills/target/
+rust/fourhill/Cargo.lock
```

Comment:
Cargo.lock generally shouldn't be excluded when target is a binary.

See https://doc.rust-lang.org/cargo/guide/cargo-toml-vs-cargo-lock.html

Reply:
Ah my bad, I must have seen that from a library crate. Fixed in 91fc5e5.